onnxruntime/featurizers_ops/cpu/mode_imputer_transformer.cc

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/common/common.h"
+#include "core/framework/data_types.h"
+#include "core/framework/data_types_internal.h"
+#include "core/framework/op_kernel.h"
+
+#include "Featurizers/ModeImputerFeaturizer.h"
+#include "Featurizers/../Archive.h"
+
+namespace onnxruntime {
+namespace featurizers {
+
+template <typename T>
```

Comment:
Do these structs need to be repeated in every source file?

Reply:
This is a part of a generated code. They are per featurizer so they may be different.

---
In reply to: [373739890](https://github.com/microsoft/onnxruntime/pull/2958#discussion_r373739890) [](ancestors = 373739890)